snuba/web/query.py

```diff
@@ -105,11 +105,15 @@ def _run_query_pipeline(
 
     storage_query_plan = dataset.get_query_plan_builder().build_plan(request)
 
+    # TODO: Break the Query Plan execution out of this method. With the division
```

Comment:
This would be an easy change to make. I could do it here, I will do it after https://github.com/getsentry/snuba/pull/860 to step on each other's feet less.